v1/topdown/cache/cache.go

```diff
@@ -103,12 +131,12 @@ func (c *Config) validateAndInjectDefaults() error {
 		}
 	}
 
-	if c.InterQueryBuiltinValueCache.MaxNumEntries == nil {
+	if c.InterQueryBuiltinValueCache.InterQueryBuiltinValueCacheConfig.MaxNumEntries == nil {
```

Comment:
Could the config field  be named just `Config` or something? Repeating the `InterQueryBuiltinValueCache` name makes this rather annoying to read 😅

Reply:
This wrapped config type has been dropped.